### DISCUSSION TITLE
issue with custom resolvers in httpx

### DISCUSSION THREAD

**Discussion by GDATTACKER-RESEARCHER** [projectdiscovery/httpx#1598] [General]
I updated my resolvers just now after thinking it's issue with resolvers but httpx is having some issue with custom resolvers when i run the tool without resolvers it works fine but after using resolvers it says ": no address found for host" for every host while i am using latest version of httpx with command "subfinder -d example.com -all | httpx  -r /root/Tools/resolvers.txt -o httpx.txt -v"

![image](https://github.com/projectdiscovery/httpx/assets/37478652/07b23b22-df83-4559-beef-985e46dee49f)

**Comment by olearycrew** [comment]
![CleanShot 2024-02-23 at 09 29 02](https://github.com/projectdiscovery/httpx/assets/6044920/3aa253e8-802a-4163-9402-735c25a1b8bc)

![CleanShot 2024-02-23 at 09 29 09](https://github.com/projectdiscovery/httpx/assets/6044920/b5620f47-fcff-4e9b-9188-49d6d105a6e6)

**Reply by olearycrew** [reply]
Are there specific resolvers you're using? Do those same resolvers work via `dig` or `nslookup` and not httpx? I would expect those to behave the same

**Comment by GDATTACKER-RESEARCHER** [comment]
I created my resolvers.txt via dnsvalidator latest

**Reply by olearycrew** [reply]
Can you share that or how you created it? I'm thinking that one or more of those resolvers isn't responding to you

**Reply by GDATTACKER-RESEARCHER** [reply]
dnsvalidator -tL https://public-dns.info/nameservers.txt -threads 20 -o resolvers.txt

**Comment by ehsandeep** [comment]
@GDATTACKER-RESEARCHER all the errored out is not resolvable hosts, do you have example of host that is valid and being failed with custom resolver input?
